hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/CreateNamespaceProcedure.java

```diff
@@ -187,6 +189,20 @@ private boolean prepareCreate(final MasterProcedureEnv env) throws IOException {
         new NamespaceExistException("Namespace " + nsDescriptor.getName() + " already exists"));
       return false;
     }
+
+    int dfsMaxComponentLength =
```

Comment:
Is this check in DFSClient or in NameNode? If it is in NameNode, we can not make sure that we have the same configuration value with NameNode?

Reply:
sorry about that，i was thinking too simplistic，i will rewrite this code